plumbing/format/idxfile/writer.go

```diff
@@ -0,0 +1,132 @@
+package idxfile
+
+import (
+	"bytes"
+	"math"
+	"sort"
+
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	"gopkg.in/src-d/go-git.v4/utils/binary"
+)
+
+type object struct {
+	hash   plumbing.Hash
+	offset int64
+	crc    uint32
+}
+
+type objects []object
```

Comment:
are we using the `objects` type for something specific (i don't see any methods for it)? If not, I'd say just go with `[]object` (or `[]Entry`)

Reply:
Nvm, saw the methods at the end of the file